stack.yaml

```diff
@@ -1,7 +1,7 @@
 # For more information, see: https://github.com/commercialhaskell/stack/blob/release/doc/yaml_configuration.md
 
 # Specifies the GHC version and set of packages available (e.g., lts-3.5, nightly-2015-09-21, ghc-7.10.2)
-resolver: lts-3.17
+resolver: lts-5.1
```

Comment:
Oof. Let's not put that in right now - if you do, I think everyone will have to rebuild all the dependencies, which will take a while.